sys/kern/dev_procstat.c

```diff
@@ -90,15 +91,17 @@ static char *get_command(proc_t *p) {
     /* omit slash */
     start += 1;
 
-  ssize_t len = strlcpy(buf, start, PROC_COMM_MAX);
-  len = min(len, PROC_COMM_MAX - 1);
+  ssize_t wanted = strlcpy(buf, start, left);
+  used += min(wanted, left);
```

Comment:
Assuming `used` doesn't include the null terminator, this line should look like this:
```suggestion
  used += min(wanted, left - 1);
```

Reply:
Yes. `used` does not include terminating null byte. In this case it doesn't matter because when `strlcpy` used whole buffer. I will include this changes, because it could be more readable.